### PR TITLE
Feature 3.non education program

### PR DESCRIPTION
**Pull request by NickBrugemann**

### CODE REVIEW

**Review by MarcoBergsma — CHANGES_REQUESTED** [review]
Nothing works anymore

**Review by MarcoBergsma — APPROVED** [review]

File: client/src/components/SelectFile.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          <input type="text">
+        </div>
+        <div>
+            <p id="fileContents"></p>
+        </div>
+    </div>
+</template>
+
+<script>
+const Papa = require('papaparse');
```
Comment (Aviumaa) [inline client/src/components/SelectFile.vue]: incontinent

File: client/src/router.ts [hunk 2]
```diff
@@ -107,6 +109,16 @@ export default new Router({
...
+    },
+    {
+      path: "/educationTasks/edit",
+      name: "editEducation",
+      component: EditEducationTask
+    },
+    {
+      path: "/nonEducationTasks/edit",
+      name: "editNonEducationTasks",
+      component: EditNonEducationTask
```
Comment (Aviumaa) [inline client/src/router.ts:123]: add guards

File: client/src/views/NonEducationTasks.vue [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <template>
   <div>
-    <tabs title="Geen Onderwijstaken"></tabs>
+    <tabs title="Geen Onderwijstaken">
+      <h1>Nick moder is dik</h1>
```
Comment (Aviumaa) [inline client/src/views/NonEducationTasks.vue]: je ben zelf dik

**Review by Aviumaa — APPROVED** [review]

File: client/src/components/SelectFile.vue (hunk 1 above)
Comment (MarcoBergsma) [reply client/src/components/SelectFile.vue]: inconsistent

**Pull request merged**